test/helpers/kubectl.go

```diff
@@ -840,7 +840,7 @@ func (kub *Kubectl) WaitForKubeDNSEntry(serviceName, serviceNamespace string) er
 
 	// If it fails we want to know if it's because of connection cannot be
 	// established or DNS does not exist.
-	digCMDFallback := "dig +tcp %s @%s"
+	//digCMDFallback := "dig +tcp %s @%s"
```

Comment:
Why is this commented out?

Reply:
It didn't seem to have any reason to exist, but I can leave it so the cleanup will come in different PR.